Makefile

```diff
@@ -72,6 +72,9 @@ CPUS ?= $(shell /bin/bash hack/read_cpus_available.sh)
 # Track code version with Docker Label.
 DOCKER_LABELS ?= git-describe="$(shell date -u +v%Y%m%d)-$(shell git describe --tags --always --dirty)"
 
+# This will force go to use the vendor files instead of using the `$GOPATH/pkg/mod`. (vendor mode)
+export GOFLAGS := -mod=vendor
```

Comment:
Can you provide more information about this flag?

Reply:
any conclusion? Must we add it?

Reply:
I have added more info about this flag in 6cc695153a217455e5ba6b8ae6055fc63d2fdd3e, and yes you must add it

Reply:
> I have added more info about this flag in [6cc6951](https://github.com/caicloud/golang-template-project/commit/6cc695153a217455e5ba6b8ae6055fc63d2fdd3e), and yes you must add it

👌